src/v2/guide/index.md

```diff
@@ -35,7 +35,7 @@ At the core of Vue.js is a system that enables us to declaratively render data t
 var app = new Vue({
   el: '#app',
   data: {
-    message: 'Hello Vue!'
+    message: 'Do you wanna build a Vue app?'
```

Comment:
Em có thắc mắc là sao chỗ này `message` mình không dùng 1 câu tiếng Việt như những ví dụ ở dưới của anh vậy anh?

Reply:
Vì reference đến Frozen:

- Do you wanna build a snowman?
- OK bye.

Reply:
Dạ, nhưng mà mình có nên reference như vậy không anh. Có thể nhiều người không hiểu ý đồ của mình và ở dưới các ví dụ nều là Vietnamese nên có thể cái này cũng vậy để thống nhất luôn.